pkg/neorpc/errors.go

```diff
@@ -30,19 +30,19 @@ const (
 const (
 	// RPCErrorCode is returned on RPC request processing error.
 	RPCErrorCode = -100
-	// RPCErrorUnknownBlockCode is returned on RPC request processing error: Unknown block
+	// RPCErrorUnknownBlockCode is returned on RPC request processing error: Unknown block.
```

Comment:
Squash this commit with the first one, they are about the same task.

Reply:
Done